### PR TITLE
Fixes to build with last SDK

### PR DESCRIPTION
**Pull request by yogh333**

### CODE REVIEW

File: ledger_app.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [app]
 build_directory = "./"
 sdk = "Rust"
-devices = ["nanos", "nanox", "nanos+", "stax", "flex"]
+devices = ["nanox", "nanos+", "stax", "flex"]
```
Comment (frol) [inline ledger_app.toml:4]: Is `nanos` not supported anymore?

File: src/app_ui/address.rs [hunk 2]
```diff
@@ -55,11 +55,22 @@ pub fn ui_display_pk_base58(public_key: &crypto::PublicKeyBe) -> Result<bool, Ap
...
+        let res = review.show(out_buf.as_str());
+        let status = NbglReviewStatus::new();
+        match res {
+            true => {
+                status.status_type(StatusType::Address).show(true);
+            }
+            false => {
+                status.status_type(StatusType::Address).show(false);
+            }
+        }
```
Comment (frol) [inline src/app_ui/address.rs]: Wouldn't it be easier to just do:

```suggestion
        NbglReviewStatus::new().status_type(StatusType::Address).show(res);
```

?

I see that this code is duplicated in a number of other places.

File: src/app_ui/sign/common/action/mod.rs [hunk 3]
```diff
@@ -303,7 +312,17 @@ pub fn ui_display_common<const N: usize>(
             last_screen = "Action confirmed";
         }
 
-        review.show("Reject", last_screen, "Transaction rejected")
+        let res = review.show("Reject");
+        let status: NbglStatus = NbglStatus::new();
+        match res {
+            true => {
```
Comment (frol) [inline src/app_ui/sign/common/action/mod.rs:311]: I feel that `if` statement is more concise and easier to read and less nesting

**Review by frol — APPROVED** [review]
Besides the question regarding `nanos` I don't have any other pressing questions. Feel free to ignore my styling suggestions

File: ledger_app.toml (hunk 1 above)
Comment (yogh333) [reply ledger_app.toml:4]: No, with the last Rust SDK Nano S cannot be supported due to limitation on stack size.

File: src/app_ui/address.rs (hunk 2 above)
Comment (yogh333) [reply src/app_ui/address.rs]: Yes indeed. Will update the code accordingly, thank you !

**Pull request merged**